pandas/tests/groupby/test_groupby.py

```diff
@@ -2190,3 +2190,18 @@ def test_groupby_mean_duplicate_index(rand_series_with_duplicate_datetimeindex):
     result = dups.groupby(level=0).mean()
     expected = dups.groupby(dups.index).mean()
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("meth", ["max", "min", "sum", "mean", "median"])
```

Comment:
hmm can you put these with similar tests, maybe in test_reductions?

Reply:
Moved